### DISCUSSION TITLE
Occasional Server Error - Class 'AttributeError' 'VirtualMachine' object has no attribute 'oob_ip_id' for certain IPs

### DISCUSSION THREAD

**Discussion by tfboy** [netbox-community/netbox#13820] [Help Wanted!] [answered]
For some IPs assigned to some VMs, I get the above error. This seems to be something new, I don't recall seeing it in previous versions, maybe I've never hit a problematic IP...

The error happens for the same IP. The URL /ipam/ip-addresses/xxxx just brings up the Server Error page.
Other IPs are fine and as far as I can tell, all IPs including the problematic ones have been assigned the same way. If I go to the VM, I can see the interface with the IP attached to it. It's just whe I click on the link of the IP address (link going to the ipam/ip-addresses/xxxx) that I get the server error.

I haven't yet deleted the IP and tried recreating it, maybe that would fix the problem. I can do that but haven't yet in case there's a troubleshooting process to help identify the root issue.

Python version 3.8.10
NetBox version 3.6.0

**Comment by jeremystretch** [comment, marked as answer]
This has been fixed in NetBox v3.6.1. Please see #13619.

**Reply by tfboy** [reply]
Thanks Jeremy. I did search, not sure why I didn't find it. Sorry!

**Comment by Flabenelli** [comment]
I experiencing the same issue:

<class 'AttributeError'>

'VirtualMachine' object has no attribute 'oob_ip_id'

Python version: 3.12.3
NetBox version: 4.1.8
Plugins: 
  netbox_attachments: 6.0.0
  netbox_dns: 1.1.6
  netbox_floorplan: 0.5.0
  netbox_lifecycle: 1.1.3
  netbox_reorder_rack: 1.1.3
  netbox_topology_views: 4.1.0

**Reply by jeremystretch** [reply]
This is _not_ related to the issue in this post, which is over a year old.

**Reply by Flabenelli** [reply]
Ok I'll create a new issue then.

**Reply by jeremystretch** [reply]
This bug has already been reported. Please see #18230.
